packages/client/wallets/aa/src/CrossmintAASDK.ts

```diff
@@ -29,16 +43,21 @@ export class CrossmintAASDK {
 
     async getOrCreateWallet<B extends BlockchainIncludingTestnet = BlockchainIncludingTestnet>(
         user: UserIdentifierParams,
-        chain: B,
+        chain: B | BackwardsCompatibleChains,
         walletConfig: WalletConfig
     ) {
         try {
+            chain = transformBackwardsCompatibleChains(chain);
             logInfo("[GET_OR_CREATE_WALLET] - INIT", {
                 service: SCW_SERVICE,
                 user,
                 chain,
             });
 
+            if (!isEVMBlockchain(chain)) {
+                throw new WalletSdkError(`The blockchain ${chain} is still not supported`);
```

Comment:
it's a bit funny as written now
```suggestion
                throw new WalletSdkError(`The blockchain ${chain} is not supported`);
```